package.json

```diff
@@ -8,53 +8,40 @@
   "author": "",
   "license": "LGPL-2.1",
   "scripts": {
-    "build": "webpack",
-    "watch": "webpack --watch",
-    "clean": "cd ./dist && rm -rf *",
-    "eslint": "eslint --ext .jsx --ext .js src/ lib/",
-    "eslint:fix": "eslint --fix --ext .jsx --ext .js src/ lib/"
+    "build": "./build.js",
+    "watch": "./build.js -w",
+    "eslint": "eslint --ext .jsx --ext .js src/",
+    "eslint:fix": "eslint --fix --ext .jsx --ext .js src/"
   },
   "devDependencies": {
-    "@babel/core": "^7.6.0",
-    "@babel/eslint-parser": "^7.13.14",
-    "@babel/preset-env": "7.12.17",
-    "@babel/preset-react": "^7.0.0",
     "argparse": "^2.0.1",
-    "babel-loader": "^8.0.0",
-    "chrome-remote-interface": "^0.32.1",
-    "compression-webpack-plugin": "^9.0.0",
-    "copy-webpack-plugin": "^9.0.0",
-    "css-loader": "^6.5.0",
-    "css-minimizer-webpack-plugin": "^3.0.1",
-    "eslint": "^7.10.0",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-config-standard-jsx": "^10.0.0",
-    "eslint-config-standard-react": "^11.0.0",
-    "eslint-plugin-flowtype": "^5.2.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "esbuild": "0.25.0",
+    "esbuild-plugin-copy": "2.1.1",
+    "esbuild-plugin-replace": "1.4.0",
+    "esbuild-sass-plugin": "3.3.1",
+    "esbuild-wasm": "0.25.0",
+    "eslint": "8.57.1",
+    "eslint-config-standard": "17.1.0",
+    "eslint-config-standard-jsx": "11.0.0",
+    "eslint-config-standard-react": "13.0.0",
+    "eslint-plugin-import": "2.31.0",
+    "eslint-plugin-jsx-a11y": "6.10.2",
+    "eslint-plugin-promise": "6.6.0",
+    "eslint-plugin-react": "7.37.4",
+    "eslint-plugin-react-hooks": "4.6.2",
+    "eslint-plugin-flowtype": "8.0.3",
```

Comment:
This can be dropped in a follow up commit, we never used flowtype annotations in cockpit.